src/_guides/language/language-tour.md

```diff
@@ -2470,9 +2470,9 @@ When exactly do assertions work?
 That depends on the tools and framework you're using:
 
 * Flutter enables assertions in [debug mode.][Flutter debug mode]
-* Development-only tools such as [dartdevc][]
+* Development-only tools such as [`webdev serve`][]
```

Comment:
This like and the compile js one below don't render correctly. I think you are missing definitions of them.

<img width="228" alt="Screen Shot 2022-08-30 at 10 49 31" src="https://user-images.githubusercontent.com/13644170/187394561-e2618c03-eac4-49c2-a345-a5ce0c138e1f.png">